ftw/builder/tests/test_portlets.py

```diff
@@ -1,10 +1,20 @@
 from ftw.builder import Builder
 from ftw.builder import create
 from ftw.builder.tests import FunctionalTestCase
-from ftw.testbrowser import browsing
 from Products.CMFPlone.utils import getFSVersionTuple
+import unittest
 
+# Disable ftw.testbrowser based tests for Plone 5.2
+# TODO: Reenable them when ftw.testbrowser is available for Plone 5.2
```

Comment:
Do we wait merging that until ftw.testbrowser is ready and these tests are updated?

Reply:
Reenabled. Tests pass with webtest branch of ftw.testbrowser